dask_pytorch/dispatch.py

```diff
@@ -0,0 +1,72 @@
+"""
+This module contains the user-facing API to submit PyTorch jobs to a Dask cluster
+"""
+
+import os
+from typing import List, Callable, Tuple
+from dask.distributed import Client
+import torch.distributed as dist
+
+
+def get_worker_info(client: Client) -> Tuple[List[str], str]:
+    """
+    returns a list of workers (sorted), and the DNS name for the master host
+    The master is the 0th worker's host
+    """
+    workers = client.scheduler_info()["workers"]
+    worker_keys = sorted(workers.keys())
+    host = workers[worker_keys[0]]["host"]
+    return worker_keys, host
+
+
+def run(client: Client, pytorch_function: Callable, *args, **kwargs):
+    """
+    Dispatch a pytorch function over a dask cluster, and returns a list of futures
+    for the resulting tasks
+    """
+    worker_keys, host = get_worker_info(client)
+    world_size = len(worker_keys)
+    port = 23456  # pick a free port?
```

Comment:
Do we want to fix this/remove this comment before release?

Reply:
no, I think it's fine